src/index.js

```diff
@@ -0,0 +1,24 @@
+import './style.css';
+
+const tasks = [{
+  description: 'wash the dishes',
+  completed: false,
+  index: 1,
+},
+{
+  description: 'complete to Do list project',
+  completed: false,
+  index: 2,
+}];
+
+const displaytasks = () => {
+  const list = document.querySelector('.list');
+  for (let i = 0; i < tasks.length; i += 1) {
+    const div = document.createElement('div');
+    div.classList.add('div');
+    div.innerHTML = `<div class="listdiv"><input type="checkbox" name="${tasks[i].description}"><label class="listlabel" for="${tasks[i].description}">${tasks[i].description}</label></div><i class="fa-solid fa-ellipsis-vertical" style="color: #acadaf"></i>`;
+    list.appendChild(div);
+  }
+};
+
+displaytasks();
```

Comment:
**OPTIONAL** - To help make your project more readable and easier to maintain, I suggest you call this function after attaching to an event listener. In this case `DOMContentLoaded`. Please fix this 🙏